client/src/socket.js

```diff
@@ -1,16 +1,20 @@
-import io from "socket.io-client";
+import { io } from "socket.io-client";
 import store from "./store";
 import {
   setNewMessage,
   removeOfflineUser,
-  addOnlineUser,
+  addOnlineUser
 } from "./store/conversations";
 
 const socket = io(window.location.origin);
 
 socket.on("connect", () => {
   console.log("connected to server");
 
+  socket.onAny((event, ...args) => {
+    console.log("onAny", event, args);
+  });
+
```

Comment:
Is this doing anything ? if not, it doesn't need to stay in the code, and we shouldn't commit console logs to our code either 🙂

Reply:
it will be removed.